test/webserv/integration/common_functions.py

```
@@ -17,6 +18,20 @@ def read_file_binary(file):
+def delete_file(file_path):
+    if file_path is None:
+        return
+
+    try:
+        if os.path.exists(file_path):
+            os.remove(file_path)
+            time.sleep(1)
+            print(f"Deleted file: {file_path}")
+    except Exception as e:
+        print(f"Error deleting file: {file_path}, {e}")
+        raise AssertionError
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)